src/NovelLoggingService.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+
+#include <iostream>
+#include "NovelLoggingService.h"
+#include "NovelRTUtilities.h"
+#include "spdlog/async.h"
+#include <typeinfo>
+
+namespace NovelRT {
```

Comment:
Do we want to adhere to the new formatting rules weve been discussing in Discord since this is a new type?

Reply:
Yes - as discussed, I indented types that come after the namespace for `NovelLoggingService`. The audio service won't be done yet since it's getting re-done anyways.